src/components/podcast/Podcast.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Episode from './episode/Episode';
+import PodcastHeader from './Header'
+import parsePodcastFeedFromXML from '../../util/podcast-util';
+
+/**
+ * Component for displaying the details about a podcast including the episodes
+ */
+const Podcast = (props) => {
+
+    // Track the podcast object
+    const [podcast, setPodcast] = useState({});
+    // Track the episodes separately
+    const [episodes, setEpisodes] = useState([])
+    // Track if the podcast has been loaded
+    const [loaded, setLoaded] = useState(false);
+
+    useEffect(() => { loadFeed(props.feed + '.xml') });
+
+    /**
+     * If the feed hasn't been loaded go out and load it.  Set the podcast
+     * state and loaded to true.
+     */
+    async function loadFeed(feedFile) {
+        if (!loaded) {
+            axios.get(feedFile)
+                .then(function (result) {
+
+
+                    const podcast = parsePodcastFeedFromXML(result.data);
+                    setPodcast(podcast);
+                    setEpisodes(podcast.episodes);
+                    setLoaded(true);
+                    if (podcast) {
+                        document.title = podcast.title;
+                    }
+                })
+        }
+    }
+
+    /** 
+     * Handle the click of the Order Flip button, which will reverse
+     * the episode list and update the state
+    */
+    const handleOrderFlip = () => {
+        episodes.reverse();
+        setEpisodes([...episodes]);
+    }
+
+
+    return (
+        <div><header><PodcastHeader podcast={podcast} /></header>
```

Comment:
style nit: I've never seen this style before. Generally every tag gets its own line.

ie:
```
<div>
  <header>
    <PodcastHeader podcast={podcast} />
  </header>
  ...
</div>
```

Reply:
Another nit: The `PodcastHeader` should most likely just use a `<header>` tag instead of a `<div>` as it's outermost wrapper.

Reply:
Moved header into the PodCastHeader component. Updated all tags so they get their own line.